pandera/api/base/schema.py

```diff
@@ -21,9 +21,9 @@
 class BaseSchema(ABC):
     """Core schema specification."""
 
-    BACKEND_REGISTRY: Dict[
-        Tuple[Type, Type], Type[BaseSchemaBackend]
-    ] = {}  # noqa
+    BACKEND_REGISTRY: Dict[Tuple[Type, Type], Type[BaseSchemaBackend]] = (
```

Comment:
is this due to updating black? would be preferable if this happened as a separate PR